### ISSUE TITLE
local variable 'papiaccess' referenced before assignment

### ISSUE THREAD

**Issue by Cozza38** [theotherp/nzbhydra#120]
Hey mate,

I just noticed I'm receiving this error when hitting the API manually with sonarr (RSS still works)

example URL: `/api?t=tvsearch&cat=5030,5040&extended=1&apikey=APIKEY&offset=0&limit=100&tvdbid=302286&season=1&ep=2`

Using the master branch.

```
2016-01-31 11:39:32,203 - ERROR - web - local variable 'papiaccess' referenced before assignment
Traceback (most recent call last):
  File "/home/hydra/nzbhydra/libs/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/hydra/nzbhydra/libs/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/hydra/nzbhydra/libs/webargs/core.py", line 368, in wrapper
    return func(*new_args, **kwargs)
  File "/home/hydra/nzbhydra/nzbhydra/web.py", line 309, in api
    return api_search(args)
  File "/home/hydra/nzbhydra/libs/flask_cache/__init__.py", line 537, in decorated_function
    rv = f(*args, **kwargs)
  File "/home/hydra/nzbhydra/nzbhydra/web.py", line 342, in api_search
    result = search.search(False, search_request)
  File "/home/hydra/nzbhydra/nzbhydra/search.py", line 180, in search
    result = search_and_handle_db(dbsearch, {x: search_request for x in indexers_to_call})
  File "/home/hydra/nzbhydra/nzbhydra/search.py", line 239, in search_and_handle_db
    results_by_indexer = start_search_futures(indexers_and_search_requests)
  File "/home/hydra/nzbhydra/nzbhydra/search.py", line 269, in start_search_futures
    results = f.result()
  File "/home/hydra/nzbhydra/libs/concurrent/futures/_base.py", line 396, in result
    return self.__get_result()
  File "/home/hydra/nzbhydra/libs/concurrent/futures/thread.py", line 55, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/hydra/nzbhydra/nzbhydra/search_module.py", line 120, in search
    queries_execution_result = self.execute_queries(urls)
  File "/home/hydra/nzbhydra/nzbhydra/search_module.py", line 323, in execute_queries
    return QueriesExecutionResult(results=results, indexerSearchEntry=psearch, indexerApiAccessEntry=papiaccess, indexerStatus=indexerStatus, total=total_results, loaded_results=len(results), total_known=total_known, has_more=has_more)
UnboundLocalError: local variable 'papiaccess' referenced before assignment
```

**Comment by theotherp** [comment]
Already fixed in develop.

**Comment by Cozza38** [comment]
Nailed it mate, Is develop something you suggest I could/should switch to?

**Comment by theotherp** [comment]
Definitely not :-) 

My workflow is a bit different from that of other teams (like sonarr) where using develop works most of the time. I commit a lot of test stuff to see if it builds on linux, if I forgot an import or what not. You would probably break your installation. I try to push to master as often as I can, but a lot of meaty changes over the last week prevented that so far.

**Comment by Cozza38** [comment]
no worries, I took a look and it absolutely destroyed my install :)

Re-cloned, restored settings and database and I'm back in action.

Thanks for the fast responses!

**Comment by theotherp** [comment]
Can you tell me how it destroyed your install?

**Comment by theotherp** [comment]
Well, let's hope it's not too bad, I'll push to master now...

**Comment by Cozza38** [comment]
Sorry mate, I replied via email but it didn't come through.

On the ui I got as far as the css not loading correctly which made everything unclickable. It was looking for the old nzbhydra.ccs which has been removed.

I did an api hit and results came back but the grab didn't appear to respect my base URL settings so I couldn't grab the nzb due to a 404.

That's about as far as I went before I reverted.
